diff --git a/lib/web/fetch/body.js b/lib/web/fetch/body.js
index 9e41f80d7e0dcff6a47987a677408fafd7c8745c..e4d603bffb94ef632584e39cad58ecf69c0587ac 100644
--- a/lib/web/fetch/body.js
+++ b/lib/web/fetch/body.js
@@ -114,7 +114,7 @@ function extractBody (object, keepalive = false) {
     source = isArrayBuffer(object)
       ? new Uint8Array(object.slice())
       : new Uint8Array(object.buffer.slice(object.byteOffset, object.byteOffset + object.byteLength))
-  } else if (webidl.is.FormData(object)) {
+  } else if (util.isFormDataLike(object)) {
     const boundary = `----formdata-undici-0${`${random(1e11)}`.padStart(11, '0')}`
     const prefix = `--${boundary}\r\nContent-Disposition: form-data`
 
